### ISSUE TITLE
[Bug]: 27.1.1 UI error

### ISSUE THREAD

**Issue by GammaPi** [nextcloud/server#40647]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

"New" button not shown completely.
![image](https://github.com/nextcloud/server/assets/19838874/8e80c732-dbd2-4f47-9eb8-77b315a751f1)
![image](https://github.com/nextcloud/server/assets/19838874/68304f07-765c-4fef-b9cf-f846b42c4903)
![image](https://github.com/nextcloud/server/assets/19838874/00b341a2-23d1-4543-a37d-e0e4a3804427)
![image](https://github.com/nextcloud/server/assets/19838874/08700270-8c7a-4e9e-8dff-63d7b64168d9)
![image](https://github.com/nextcloud/server/assets/19838874/acd29817-344b-461a-ba17-b235285f5f7f)
![image](https://github.com/nextcloud/server/assets/19838874/a3f98331-688a-4677-83ae-dee4143a2bdf)
![image](https://github.com/nextcloud/server/assets/19838874/4cf00a09-dc52-441b-bf92-ca8de9290efb)

### Steps to reproduce

1. docker pull nextcloud:stable-apache
2. Open browser

### Expected behavior

UI should display normally

### Installation method

Community Docker image

### Nextcloud Server version

27

### Operating system

Debian/Ubuntu

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

Upgraded to a MAJOR version (ex. 22 to 23)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
I tried to disable every app and it doesn't work
```

### Nextcloud Signing status

```shell
No errors have been found.
```

**Comment by GammaPi** [comment]
I tried:
```
docker run -d -p 127.0.0.1:8080:80 nextcloud:stable-apache
```
and it does not work out of the box.

**Comment by Ornanovitch** [comment]
I had the same problem: I had to logout from my account and to login again to get it right.

**Comment by blizzz** [comment]
This is a regression in 27.1.1 and already fixed for 27.1.2. RC is out, final planned for next week.
